rust/src/stats.rs

```diff
@@ -0,0 +1,76 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use crate::format_one_arg;
+use reqwest::header::CONTENT_TYPE;
+use reqwest::header::USER_AGENT;
+use reqwest::Client;
+use serde::{Deserialize, Serialize};
+use std::sync::mpsc::Sender;
+use std::time::Duration;
+
+const PLAUSIBLE_URL: &str = "https://plausible.io/api/event";
+const SM_USER_AGENT: &str = "Selenium Manager {}";
+const APP_JSON: &str = "application/json";
+const PAGE_VIEW: &str = "pageview";
+const SELENIUM_DOMAIN: &str = "manager.selenium.dev";
+const SM_STATS_URL: &str = "https://{}/sm-stats";
+const REQUEST_TIMEOUT_SEC: u64 = 3;
+
+#[derive(Default, Serialize, Deserialize)]
+pub struct Data {
+    pub name: String,
+    pub url: String,
+    pub domain: String,
+    pub props: Props,
+}
+
+#[derive(Default, Serialize, Deserialize)]
+pub struct Props {
+    pub browser: String,
+    pub browser_version: String,
+    pub os: String,
+    pub arch: String,
+    pub lang: String,
+    pub selenium_version: String,
```

Comment:
Can we lower case all these properties, please? I just tried it and we get different values if they are sent with different casing.

Reply:
e.g.

<img width="1130" alt="image" src="https://github.com/SeleniumHQ/selenium/assets/5992658/0c919d42-81a6-4293-806c-36b171af32af">

Reply:
> Can we lower case all these properties, please?

I have committed a patch to send them in lowercase.

Reply:
Great! Let's merge this and I will implement the extra flag in the different languages.